.github/workflows/branch_test_push.yml

```diff
@@ -10,6 +10,8 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@master
+        with:
+          fetch-depth: '0'
```

Comment:
Any particular reason for needing the full history when checking out the repo in this action workflow?

Reply:
I see now, necessary for proper date extraction in tests. Added to both workflows.